eng/Version.Details.xml

```diff
@@ -6,12 +6,32 @@
       <Sha>525b6c35cc5c5c9b80b47044be2e4e77858d505a</Sha>
       <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
     </Dependency>
-    <!-- Necessary for source-build. This allows the packages to be retrieved from previously-source-built artifacts
+    <!-- Necessary for source-build. This allows the package to be retrieved from previously-source-built artifacts
+      and flow in as dependencies of the packages produced by msbuild. -->
```

Comment:
It'd be nice if the comment said it was a transitive dependency of S.R.MetadataLoadContext, so we know when we can remove it.

Reply:
@mthalman - can you add the suggested comment (or not - if you thing it's unnecessary) and resolve this conversation? We currently require all conversations to be resolved (to prevent accidentally missed communications) before merging. Once this is resolved the PR will auto-merge

Reply:
Thanks for the reminder.